### DISCUSSION TITLE
Validate yaml frontmatter in a markdown file

### DISCUSSION THREAD

**Discussion by wmeints** [microsoft/vscode-discussions#1560] [Extension Development QnA]
I'm developing an extension that uses markdown with frontmatter to create test cases. I tried configuring the extension to recognize `.prompt` files as markdown files. Now, I want the front matter of the file to follow a particular schema. Is there a way to make this work with a grammar or something similar? I tried to set up things with embedded languages and injection grammar, but I'm not getting it to work properly.

**Comment by isidorn** [comment]
Do you still have this issue? Can you share some minimal repro steps?

**Reply by olguzzar** [reply]
Hi @wmeints, let us know if you have further repro steps so we can assist. If we haven't heard any new information in the next week, we'll close this discussion due to inactivity - thanks!

**Reply by wmeints** [reply]
I still have the issue, but I didn't find the time to look at it some more. I'll create a new discussion when I have the time to look at it some more.
